lib/web/server_helpers.rb

```diff
@@ -118,4 +118,17 @@ def query_to_boolean(value, default: false)
   def table_reload_timing
     Inventory.load("local_setting")["webui.table.reload-timing"] || RELOAD_TIMING_DEFAULT
   end
+
+  def partial(template, *args)
+    template_file_name = "_#{template}".intern
+    options = args.last.is_a?(Hash) ? args.pop : {}
+    options.merge!(layout: false)
+    if collection = options.delete(:collection) then
```

Comment:
__[RuboCop]__

Lint/AssignmentInCondition: Use `==` if you meant to do a comparison or wrap the expression in parentheses to indicate you meant to assign in a condition.


<!-- Comment by [Sider](https://sider.review) -->